_articles/open-firmware-smmstore.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: article
+title: Firmware Flash to Clear the SMMSTORE ROM (Error code 5)
+description: >
+  How to fix the failed with exit status: exit code: 5 message
+keywords:
+  - Firmware
+  - Open Firmware
+  - SMMSTORE
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: software-applications
+
+---
+
+## Things that you will need
+
+- 1-2 flash drives (1GB and up)
+- Philips Screwdriver
+- Approx. 30 mins.
+
+## The error messages
+
+If you are seeing any of these messages on your Open Firmware system:
+
+```
+configuring bootloader error: command failed with exit status: exit code: 5
+```
+
+```
+errored while installing system: command failed with exit status: exit code: 5
+```
+
+```
+calling Schedule method failed: "failed to add boot entry: exit code: 5
+```
+
+```
+system76-firmware: failed to schedule: failed to add boot entry: exit code: 5
+```
+
+we may need to clear the SMMSTORE ROM by re-flashing a patched firmware using a flash drive with a custom firmware file. Here are the steps to perform that action. After flashing, this will remove any custom EFI variables (such as those installed by boot managers). We have information on repairing the bootloader at the end of this article. 
```

Comment:
The firmware file itself is unmodified. The updater is modified it not read/write EFI variables.

Reply:
Would it be 'by re-flashing the firmware using a..." then?

Reply:
This should only be needed for upgrades from firmware older than 2021-01-19_d6d37c0. That version and newer have addressed the issues, and provide a different way to clear the SMMSTORE (by removing the CMOS) if needed.

So it should read something like:

> we may need to clear the SMMSTORE ROM by flashing new firmware using a flash drive with a custom firmware update tool

Reply:
Fixed in this commit : https://github.com/system76/docs/pull/561/commits/52af5ce7c13f97413aab4d7cf42d29bf1c3724e5